com.unity.render-pipelines.universal/Documentation~/rendering/deferred-rendering-path.md

```diff
@@ -0,0 +1,422 @@
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>AfterRenderingPostProcessing</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tr>
+        <td>AfterRendering</td>
+        <td>&#160;</td>
+        <td>&#160;</td>
+    </tr>
+    <tbody>
+</table>
+
+The following sections describe the Deferred Rendering Path render Passes.
+
+#### Depth, or depth and normal prepass
+
+In the Deferred Rendering Path, in the depth prepass or depth and normal prepass, Unity renders only the Materials that do not support the deferred rendering model. For example, Materials using the Complex Lit shader are such Materials.
+
+In the Deferred Rendering Path, Unity does not use the depth prepass to generate a copy of the depth buffer (this behavior is different in the Forward Rendering Path).
+
+If the Universal Renderer has the SSAO Renderer Feature, Unity executes the depth and normal prepass. SSAO uses the screen-space depth and normal buffers to calculate ambient occlusion.
+
+#### Optional passes: SSAO, SSAO with blending
+
+If the Universal Renderer has the SSAO Renderer Feature, and the **After Opaque** option is disabled (the option is disabled by default), Unity executes the SSAO Pass at the AfterRenderingGbuffer event. The SSAO Renderer Feature calculates the SSAO texture. Unity samples this texture in the Deferred rendering Pass and in the Pass that renders Forward-only Materials.
+
+Using this Pass order, Unity can combine the baked occlusion with the real-time occlusion from the SSAO Renderer Feature, and avoid double-darkening from the baked and the real-time ambient occlusion.
+
+When the After Opaque option is enabled, Unity executes the SSAO and blending Pass at the AfterRenderingOpaques event, after rendering the Forward-only Materials. Unity then executes an extra full screen Pass to overlay the SSAO texture onto the Emissive/GI/Lighting buffer. This causes over-darkening of the areas that receive baked occlusion and real-time occlusion.
+
+**Performance considerations**
+
+On mobile platforms with TBDR architecture, with the **After Opaque** option is disabled, Unity requires an extra render target for load and store operations. This has a significant performance impact.
+
+Enabling the **After Opaque** option on mobile platforms improves GPU performance. On mobile platforms with TBDR architecture, enabling this option avoids extra render target load and store operations.
+
+#### Forward-only Pass
+
+Certain Unity shaders use lighting models that Unity cannot render in the Deferred Rendering Path.
+
+Examples of such shaders:
+
+* **Complex Lit**: the Lighting model of this shader (for example, the Clear Coat effect) is too complex and extra Material properties cannot fit into the G-buffer.
+
+* **Baked Lit** and **Unlit**: these shaders do not receive dynamic lighting, so writing them directly into the Lighting buffer instead of the G-buffer is faster.
```

Comment:
Isn't the `Lighting` buffer the `Emissive/GI/Lighting` buffer described in G-buffer layout section?
I guess you mean `BakedLit` and `Unlit` are written directly to Lighting buffers instead of being evaluated in Stencil Lighting pass as it's faster?

Reply:
> BakedLit and Unlit are written directly to Lighting buffers instead of being evaluated in Stencil Lighting pass
You are correct.

@oleks-k  maybe rewrite as something like this?
* **Baked Lit** and **Unlit**: these shaders do not calculate realtime lighting, so rendering them directly into the Lighting buffer during forward-only pass is faster.

Reply:
Thanks! @kaychang-unity , take a look at the new text please, is it correct?

Reply:
Hi, sorry after re-reading again, what I originally wrote is a bit ambiguous. Can you edit again (here or later PR):
- **Baked Lit and Unlit**: these shaders do not receive dynamic lighting, so writing them directly into the Lighting buffer during forward-only pass instead of the G-buffer pass is faster.

Reply:
@kaychang-unity , thanks for the input, I'll move this to a separate task as well, I'd like to understand the concepts in this section more and then edit the description.